demos/http/http_demo_s3_download/http_demo_s3_download.c

```diff
@@ -142,23 +253,49 @@ static HTTPRequestInfo_t requestInfo;
 static HTTPResponse_t response;
 
 /**
- * @brief The host address string extracted from the pre-signed URL.
+ * @brief The host address string extracted from the given server URL.
```

Comment:
I don't see a URL being provided in the `demo_config.h`. This needs to be updated

Reply:
Same for other place where "given server URL" is used